mobile/mysterium/entrypoint.go

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package mysterium
+
+import (
+	"path/filepath"
+
+	log "github.com/cihub/seelog"
+	"github.com/mitchellh/go-homedir"
+	openvpn_core "github.com/mysteriumnetwork/go-openvpn/openvpn/core"
+	"github.com/mysteriumnetwork/node/cmd"
+	"github.com/mysteriumnetwork/node/core/connection"
+	"github.com/mysteriumnetwork/node/core/node"
+	"github.com/mysteriumnetwork/node/identity"
+	"github.com/mysteriumnetwork/node/metadata"
+	service_noop "github.com/mysteriumnetwork/node/services/noop"
+)
+
+// NewNode function creates new Node
+func NewNode() {
+	var di cmd.Dependencies
+
+	userHomeDir, err := homedir.Dir()
```

Comment:
this requires additional external dependency for non-Darwin os'es:
`cmd := exec.Command("getent", "passwd", strconv.Itoa(os.Getuid()))` 
not sure its proper way to do it.